BookingsApi/BookingsApi/Services/HearingParticipantService.cs

```diff
@@ -291,4 +254,34 @@ private async Task UpdateHearingStatusAsync(Guid hearingId, BookingStatus bookin
         var command = new UpdateHearingStatusCommand(hearingId, bookingStatus, updatedBy, cancelReason);
         await _commandHandler.Handle(command);
     }
+    
+    private static ExistingParticipantDetails UpdateExistingParticipantDetailsFromRequest<T>(List<Participant> existingParticipants, T existingParticipantRequest) where T : IUpdateParticipantRequest
+    {
+        var existingParticipant = existingParticipants.SingleOrDefault(ep => ep.Id == existingParticipantRequest.ParticipantId);
+        var isNotBeingUpdated = !CheckParticipantIsBeingUpdated(existingParticipant, existingParticipantRequest);
+            
+        if (existingParticipant == null || isNotBeingUpdated)
+            return null;
+            
+        return new ExistingParticipantDetails
+        {
+            DisplayName = existingParticipantRequest.DisplayName,
+            OrganisationName = existingParticipantRequest.OrganisationName,
+            ParticipantId = existingParticipantRequest.ParticipantId,
+            Person = existingParticipant.Person,
+            RepresentativeInformation = new RepresentativeInformation { Representee = existingParticipantRequest.Representee },
+            TelephoneNumber = existingParticipantRequest.TelephoneNumber,
+            Title = existingParticipantRequest.Title
+        };
+    }
+    
+    //Check the only properties that can be updated on an existing Participant have been edited, otherwise should be excluded from updates to the video-api
+    private static bool CheckParticipantIsBeingUpdated(Participant existingParticipant, IUpdateParticipantRequest requestChanges)
+    {
+        return existingParticipant?.Person?.Title != requestChanges.Title ||
+               existingParticipant?.Person?.Organisation?.Name != requestChanges.OrganisationName ||
+               existingParticipant?.Person?.TelephoneNumber != requestChanges.TelephoneNumber ||
+               existingParticipant?.DisplayName != requestChanges.DisplayName;
```

Comment:
do we need to check the Representee property as well?

Reply:
I don't think that can be updated once the booking has been made. These fields are based on that admin web form